tsconfig.json

```diff
@@ -1,21 +1,26 @@
 {
   "compilerOptions": {
     "target": "ES2020",
-    "useDefineForClassFields": true,
     "lib": ["ES2020", "DOM", "DOM.Iterable"],
     "module": "ESNext",
     "skipLibCheck": true,
     "useUnknownInCatchVariables": true,
     "strict": true,
+    "esModuleInterop": true,
```

Comment:
Lets discuss the reasons for these config changes and check if they are necessary, I'd be hesitant to make any changes to the config without good reason